### PR TITLE
Refactor twitter feed banner

### PR DESCRIPTION
**Pull request by natmunckton**

### CODE REVIEW

File: assets/js/entry.js [hunk 1]
```diff
@@ -2,10 +2,12 @@ import {isModernBrowser, loadScript, isDevEnvironment} from './modules/utilities
 import kcc from './modules/legacy';
 import * as globals from './globals';
 import LazyLoader from './modules/lazyloader';
+import TwitterFeed from './modules/twitter-feed';
 
 function main() {
   kcc.init();
   LazyLoader.loadScripts();
+  let twitterFeed = new TwitterFeed().init();
```
Comment (wellhairy) [inline assets/js/entry.js]: is this how we're leaving this or did we decide to load this in diffrently

File: assets/scss/partials/elements/_twitter-feed.scss [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    padding-right: 30px;
+    font-size: 18px;
+    line-height: 1.2;
+  }
+
+  @include breakpoint(medium) {
+    min-height: 99px;
+    padding: 11px 15px 15px;
+
+    &__profile-box {
```
Comment (wellhairy) [inline assets/scss/partials/elements/_twitter-feed.scss]: I'm a little concerned just looking at the rectangular nature of this element that it's been made with specifically the KCC logo in mind and not taken any other campaigns like for instance highways into account.

File: assets/scss/partials/elements/_twitter-feed.scss [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      #{$this}__time {
+        font-size: 18px;
+      }
+
+      #{$this}__message {
+        font-size: 22px;
+      }
+    }
+  }
+}
```
Comment (wellhairy) [inline assets/scss/partials/elements/_twitter-feed.scss]: Id also ask why the bird was taken out from the background. however i'm not sure if i care all that much i do have to wonder if it looses some of its character

File: assets/scss/partials/structure/_cards.scss [hunk 4]
```diff
@@ -67,8 +67,8 @@
   }
 
   &--outline {
+    background: $insipid-white;
```
Comment (wellhairy) [inline assets/scss/partials/structure/_cards.scss]: This also seems like a bit of a major change to a card modifier named outline. I possibly wouldn't expect a background colour.

File: assets/js/modules/utilities.js [hunk 5]
```diff
@@ -75,6 +75,19 @@ export function isDevEnvironment() {
     (document.domain === 'localhost');
 }
 
+export function convertDateToRelative(date) {
+  let delta = parseInt((Date.now() - date.getTime()) / 1000);
+  let months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
```
Comment (wellhairy) [inline assets/js/modules/utilities.js]: Could this go in globals

Comment (wellhairy) [inline assets/js/modules/utilities.js]: You might need to review the const and let vars as well

File: assets/js/modules/utilities.js [hunk 6]
```diff
@@ -75,6 +75,19 @@ export function isDevEnvironment() {
...
+  let shortDate = `${date.getDate()} ${months[date.getMonth()]}`;
+
+  return delta < 60 ? `just now` :
+    delta < 120 ? `1 min ago` :
+    delta < 3600 ? `${parseInt(delta / 60)} mins ago` :
+    delta < 7200 ? `1 hour ago` :
+    delta < 86400 ? `${parseInt(delta / 3600)} hours ago` :
```
Comment (wellhairy) [inline assets/js/modules/utilities.js]: can we make these integers into variables for readability

**Comment by wellhairy** [comment]
We need to re-open this request after problematic merge

File: assets/scss/partials/elements/_twitter-feed.scss (hunk 2 above)
Comment (natmunckton) [inline assets/scss/partials/elements/_twitter-feed.scss]: Agree this is a problem – resolved by removing all logos in Twitter feed

File: assets/scss/partials/elements/_twitter-feed.scss (hunk 3 above)
Comment (natmunckton) [inline assets/scss/partials/elements/_twitter-feed.scss]: With the removal of the KCC logo and the increased size and prominence of the blue Twitter bird, the background bird now looks extraneous and repetitious.

File: assets/scss/partials/structure/_cards.scss (hunk 4 above)
Comment (natmunckton) [inline assets/scss/partials/structure/_cards.scss]: Agree with this:  --outline modifier now has a mid-grey border only.

**Pull request merged**